packages/augur-ui/src/modules/alerts/reducers/alerts.ts

```diff
@@ -37,7 +37,7 @@ const DEFAULT_STATE: Array<Alert> = [];
  * @returns {Alert[]}
  *
  */
-export default function(alerts = DEFAULT_STATE, { data, type }: BaseAction): Array<Alert> {
+export default function alert(alerts = DEFAULT_STATE, { data, type }: BaseAction): Array<Alert> {
```

Comment:
is adding `alert` here a mistake?

Reply:
Nope, as it was written this will make a webpack build fail. Justin and I were unsure of how master was even passing with this.

Reply:
there are many `export default function` methods in the Ui. do they all need to change?

Reply:
Doesn't seem so. @justinbarry discovered and fixed this so he can likely speak to it better

Reply:
I really have no idea why this is the case. Possible the babel loader is pulling in a config from one of the required packages but that is just a guess.

Reply:
I just confirmed master is building fine. This is def one of those things we can tackle later.